addons/website_event/static/src/js/website_event.js

```diff
@@ -12,17 +11,27 @@ var EventRegistrationForm = publicWidget.Widget.extend({
      */
     start: function () {
         var self = this;
+        var post = this._getPost()
```

Comment:
You could use `const` instead of `var` as the `post` value isn't modified. :slightly_smiling_face: 
Missing `;` at end of line